skyfield/almanac.py

```diff
@@ -230,9 +370,85 @@ def moon_phase_at(t):
     moon_phase_at.rough_period = 7.0  # one lunar phase per week
     return moon_phase_at
 
+
 def _distance_to(center, target):
     def distance_at(t):
         t._nutation_angles = iau2000b(t.tt)
         distance = center.at(t).observe(target).distance().au
         return distance
     return distance_at
+
+
+def satellite_visible(ephemeris, topos, satellite, alt_deg_thresh=0):
```

Comment:
These are a useful collection of routines! I wonder if they go beyond what an "almanac" is, though. Might they belong in a module of their own? For example, we could put them in the sgp4 module as one possible place for them to live?

Reply:
(And we don't need to answer that immediately — I can always shuffle things around before the next release, after accepting this PR — but I thought I'd ask to hear your ideas on the question!)

Reply:
Yeah I thought about that too. I didn't really know where to put the code so I decided for the closest one in terms of tasks